docs/content/tutorials/nginx-ingress-istio.md

```diff
@@ -0,0 +1,290 @@
+---
+title: NGINX Ingress Controller and Istio Service Mesh
+description: |
+  Use NGINX Ingress Controller with Istio Service Mesh.
+weight: 1800
+doctypes: ["concept"]
+toc: true
+docs: "DOCS-889"
+---
+
+You can use NGINX Ingress Controller fpr applications that are running inside an Istio service mesh. This allows to continue using the advanced capabilities that NGINX Ingress Controleler in a istio service mesh environments without resorting to any workarounds. 
+
+
+Prior to 1.11 release, a configuration below would send two host headers to the backend; 

```

Comment:
This section does not fit well here.
The explanation of the prior release does not relate to the conversation.
It might be useful at the end for additional technical depth.
The beginning should focus on the what and why.